src/ansys/geometry/core/misc/unsupported.py

```diff
@@ -0,0 +1,268 @@
+# Copyright (C) 2023 - 2025 ANSYS, Inc. and/or its affiliates.
```

Comment:
This entire module shouldn't be here... It should go inside the ``tools`` subpackage IMO.